File: hellojmesurfaceview/src/main/java/com/jme3/hellojmesurfaceview/MyGame.java
package com.jme3.hellojmesurfaceview;

import com.jme3.app.LegacyApplication;
import com.jme3.app.SimpleApplication;
import com.jme3.asset.AssetKey;
import com.jme3.material.Material;
import com.jme3.math.ColorRGBA;
import com.jme3.scene.Geometry;
import com.jme3.scene.shape.Sphere;

/**
 * <b>Your Actual Jme Game class.</b>
 * <br>
 * To use it inside Android :
 * <ol>
 * <li>Create an instance of it inside the #{@link MainActivity}</li>
 * <li>Set that instance using #{@link com.jme3.app.jmeSurfaceView.JmeSurfaceView#setLegacyApplication(LegacyApplication)}</li>
 * <li>Start the game using #{@link com.jme3.app.jmeSurfaceView.JmeSurfaceView#startRenderer(int)}</li>
 * </ol>
 * @author pavl_g
 */
public class MyGame extends SimpleApplication {
    @Override
    public void simpleInitApp() {
        Sphere mySphere=new Sphere(10,50,50);
        Geometry geometry = new Geometry("ball", mySphere);
        geometry.setLocalScale(0.05f);
        Material material = new Material(assetManager.loadAsset(new AssetKey<>("Common/MatDefs/Misc/Unshaded.j3md")));
        material.setColor("Color", ColorRGBA.randomColor().mult(2f));
        geometry.setMaterial(material);
        rootNode.attachChild(geometry);
    }
}

File: hellokotlin/src/main/java/com/scrappers/hellokotlin/MyGame.kt
package com.scrappers.hellokotlin

import com.jme3.app.SimpleApplication
import com.jme3.asset.AssetKey
import com.jme3.material.Material
import com.jme3.math.ColorRGBA
import com.jme3.scene.Geometry
import com.jme3.scene.shape.Sphere

/**
 * **Your Actual Jme Game class.**
 * <br></br>
 * To use it inside Android :
 *
 *  1. Create an instance of it inside the #[MainActivity]
 *  1. Set that instance using #[com.jme3.app.jmeSurfaceView.JmeSurfaceView.setLegacyApplication]
 *  1. Start the game using #[com.jme3.app.jmeSurfaceView.JmeSurfaceView.startRenderer]
 *
 *
 * @author pavl_g
 */
class MyGame : SimpleApplication() {
    override fun simpleInitApp() {
        flyByCamera.isDragToRotate = false
        val mySphere = Sphere(10, 50, 50F)
        val geometry = Geometry("ball", mySphere)
        geometry.setLocalScale(0.05f)
        val material =
            Material(assetManager.loadAsset(AssetKey("Common/MatDefs/Misc/Unshaded.j3md")))
        material.setColor("Color", ColorRGBA.randomColor().mult(2f))
        geometry.material = material
        rootNode.attachChild(geometry)
    }
}